src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -355,14 +350,14 @@ private static float strength(final Collection<Unit> units, final boolean attack
    *
    * @param blitzHere
    *        - Territory expecting to be blitzed
-   * @param blitzTerr
+   * @param blitzTerrRoutes
    *        - Territory which is being blitzed through (not guaranteed to be all possible route territories!)
```

Comment:
I think this comment applied to `blitzTerr`; it doesn't seem to make sense for `blitzTerrRoutes`.  I would just remove it if you can't determine an accurate description for `blitzTerrRoutes`.

Reply:
Good catch, this happened because I was iterating and had enough context switches I eventually assumed the variable name was mispelled and moved quickly on : )

Since the param is now gone, I removed the javadoc @param description to match.